### PR TITLE
Add stars to rating

### PR DESCRIPTION
**Pull request by ghost**
### What we did

- Add visible stars in show.html.haml
- Add step definition for showing stars on page
- Add scenario for bad rating error handler

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -34,3 +34,6 @@ coverage/public/packs
 yarn-debug.log*
 .yarn-integrity
 package-lock.json
+
+
+*.pdf
```
Comment (koggs) [inline .gitignore]: is this pdf related to raiting?

File: app/controllers/ratings_controller.rb [hunk 2]
```diff
@@ -1,20 +1,20 @@
-class RatingsController < ApplicationController 
+class RatingsController < ApplicationController
 
   before_action :auth_user
 
   def create
+    # binding.pry
```
Comment (koggs) [inline app/controllers/ratings_controller.rb]: remove this binding.pry

File: app/assets/images/icons/full_star.svg [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+</svg>
```
Comment (koggs) [inline app/assets/images/icons/full_star.svg]: is this folder being used?

File: app/assets/images/icons/empty_star.svg [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+</svg>
```
Comment (koggs) [inline app/assets/images/icons/empty_star.svg]: is this folder being used?

**Review by koggs — CHANGES_REQUESTED** [review]

File: app/controllers/ratings_controller.rb (hunk 2 above)
Comment (ghost) [reply app/controllers/ratings_controller.rb]: Removed

File: app/assets/images/icons/full_star.svg (hunk 3 above)
Comment (ghost) [reply app/assets/images/icons/full_star.svg]: This file is not being used, it has been removed

File: app/assets/images/icons/empty_star.svg (hunk 4 above)
Comment (ghost) [reply app/assets/images/icons/empty_star.svg]: This file is not being used, it has been removed

File: .gitignore (hunk 1 above)
Comment (tochman) [reply .gitignore]: No, I had a lot of pdf generated in my root folder  and I wanted to ignore them

File: app/assets/javascripts/application.js [hunk 5]
```diff
@@ -14,3 +14,18 @@
...
+    } else if( isNaN(parseInt(Number(this.valueOf()))) ) {
+      throw new TypeError("Object is not a valid number");
+    }
+    for (var i = 0; i < Number(this.valueOf()); i++) {
+      callback(i);
+    }
+  };
+  
+  String.prototype.times = timesFunction;
+  Number.prototype.times = timesFunction;
```
Comment (tochman) [inline app/assets/javascripts/application.js]: I would like to to refactor this to es6 and format the code.

**Review by magnus-thor — APPROVED** [review]
Looks good
